pyproject.toml

```diff
@@ -36,6 +36,7 @@ pytest-cov = "^6.2.1"
 pytest = "^8.4.1"
 requests-mock = "^1.12.1"
 setuptools = "^80.9.0"
+httpx = "^0.28.1"
```

Comment:
Might be missing but is this used anywhere?

Reply:
Maybe I was wrong
https://github.com/openwallet-foundation/acapy-vc-authn-oidc/actions/runs/16451908347/job/46499061876?pr=802